WireAPI/Package.swift

```diff
@@ -5,7 +5,8 @@ import PackageDescription
 let package = Package(
     name: "WireAPI",
     platforms: [
-        .iOS(.v15)
+        .iOS(.v15),
+        .macOS(.v12)
```

Comment:
This was a quick solution for the issue from App Store Connect this morning and discussed with @netbe and @johnxnguyen :
<img width="762" alt="Screenshot 2024-05-22 at 11 53 31" src="https://github.com/wireapp/wire-ios/assets/7196958/fde55619-607f-4ead-b839-c201a441885d">